.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Adopt ruff linter
+70370adaec7c75cb69b5e2a41bc5a6aa1d6b273f
```

Comment:
This SHA doesn't appear to exist. Does it need to be updated?

Reply:
Yes this is because I squashed so we can have a single commit/sha

Reply:
OK, see https://github.com/scipy/scipy/pull/17897.